rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java

```diff
@@ -573,7 +569,7 @@ public NodeSourceDescriptor updateDynamicParameters(List<Serializable> infrastru
         return this.descriptor;
     }
 
-    public void reconfigure(Object[] updatedInfrastructureParams, Object[] updatedPolicyParams) {
+    public void reconfigure(Object[] updatedInfrastructureParams, Object[] updatedPolicyParams) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)